### PR TITLE
[bitnami/mongodb] topology override option

### PR DESCRIPTION
**Pull request by kazcrn-omnisend**
### Description of the change

Enables to override (anti-)affintiy topology key in common lib

### Benefits

Enables to override (anti-)affintiy topology key in common lib

### Possible drawbacks

None

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mongodb/Chart.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     tags:
       - bitnami-common
-    version: 2.x.x
+    version: 2.2.x
```
Comment (carrodher) [inline bitnami/mongodb/Chart.yaml:10]: We usually use the major version here
```suggestion
    version: 2.x.x
```

File: bitnami/mongodb/values.yaml [hunk 2]
```diff
@@ -75,6 +75,11 @@ commonLabels: {}
 ##
 commonAnnotations: {}
 
+## @param topologyKey Uncomment following line to overide default topology key value from common lib
+##
+## topologyKey: topology.kubernetes.io/zone
+topologyKey: ""
```
Comment (carrodher) [inline bitnami/mongodb/values.yaml:81]:
```suggestion
## @param topologyKey Uncomment the following line to override the default topology key value from the common lib
## i.e topologyKey: topology.kubernetes.io/zone
##
topologyKey: ""
```

File: bitnami/mongodb/Chart.yaml [hunk 3]
```diff
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mongodb
   - https://mongodb.org
-version: 13.5.0
+version: 13.5.1
```
Comment (carrodher) [inline bitnami/mongodb/Chart.yaml:29]: Since it's a new feature addition, I would bump the minor version
```suggestion
version: 13.6.0
```

**Review by carrodher — COMMENTED** [review]
Thanks for your contribution, please take a look at the inline comments

File: bitnami/mongodb/values.yaml (hunk 2 above)
Comment (kazcrn-omnisend) [reply bitnami/mongodb/values.yaml]: I have rewritten this part to different sentence

**Comment by kazcrn-omnisend** [comment]
Hi @carrodher,
Updated according you comments

File: bitnami/mongodb/values.yaml [hunk 4]
```diff
@@ -75,6 +75,11 @@ commonLabels: {}
 ##
 commonAnnotations: {}
 
+## @param topologyKey Override common lib default topology key. If empty - "kubernetes.io/hostname" is used
+##
+## i.e. topologyKey: topology.kubernetes.io/zone
+topologyKey: ""
```
Comment (carrodher) [inline bitnami/mongodb/values.yaml:81]:
```suggestion
## @param topologyKey Override common lib default topology key. If empty - "kubernetes.io/hostname" is used
## i.e. topologyKey: topology.kubernetes.io/zone
##
topologyKey: ""
```

**Comment by kazcrn-omnisend** [comment]
Hi @carrodher ,
I have confirmed your suggested changes

**Comment by aoterolorenzo** [comment]
Hi @kazcrn-omnisend,

Since was @carrodher who have requested the changes and proceeded to the review I was just waiting for him to accept them, but seems he's on vacation with several days still remaining so I will approve the PR now in his behalf, since it seems good enough to me.

**Comment by kazcrn-omnisend** [comment]
Thank you :)

**Pull request merged**